peps/pep-9999.rst

```diff
@@ -105,27 +115,83 @@ interval math or numerical integration.
 Other proposed names include ``ntheory`` (like SymPy's submodule),
 ``integermath`` and ``imaths``.
 
+Module scope and possible library members
+=========================================
```

Comment:
I think this should be part of the "Open Issues" section.

Reply:
I think it's not an open issue because we don't want the SC deciding which library members will be added.  This is just a way of illustrating scope.  We're not looking to start a debate about the choices in this section.

Open issues are for things that the SC needs to decide.  Like the name of the library.